lightwood/api/json_ai.py

```diff
@@ -627,7 +627,7 @@ class Predictor(PredictorInterface):
     mode: str
 
     def __init__(self):
-        seed()
+        seed({json_ai.problem_definition.seed_nr})
```

Comment:
is json_ai global? How is this inherited? Shouldn't it be set from self?

Reply:
No... this is happening in `json_ai.py` when we are constructing the code for the Predictor class.

Values specified in the problem_definition get inlined in the Predictor class.

I'm not sure the currently built predictor classes are the most readable... but the current questions we are focusing on are more towards json ai design, so I'd rather tackle the generated predictors afterwards.

Reply:
got it